blocks/card/card.js

```diff
@@ -0,0 +1,41 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+  const card = document.createElement('div');
+  card.className = 'card-box';
```

Comment:
Is there a reason we went with `card-box` instead of `card`? If not, let's use just `card`.